synthesizer/src/vm/finalize.rs

```diff
@@ -791,17 +792,66 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                     //     }
                     // }
 
-                    // Initialize the stakers.
-                    let mut stakers = IndexMap::with_capacity(committee.members().len());
-                    // Iterate over the committee members.
-                    for (validator, (microcredits, _)) in committee.members() {
-                        // Insert the validator into the stakers.
-                        stakers.insert(*validator, (*validator, *microcredits));
+                    // Ensure that all members of the committee are open.
+                    ensure!(
+                        committee.members().iter().all(|(_, (_, is_open))| *is_open),
+                        "Ratify::Genesis(..) all members of the committee must be open"
+                    );
+
+                    // Calculate the stake per validator using `bonded_balances`.
+                    let mut stake_per_validator = IndexMap::with_capacity(committee.members().len());
```

Comment:
Rename this to `stakers`, why was this changed? Add back:
```
// Initialize the stakers.
let mut stakers = IndexMap::with_capacity(committee.members().len());
```

Reply:
This stakers (which can be both validators and delegators) is provided in `bonded_balances`, which is checked for consistency against the `committee` using `stake_per_validator`.
We could consider renaming `bonded_balances` to `stakers` for more clarity.
We could also consider removing some of the redundant information between `committee` and `bonded_balances`.